packages/core-components/src/lib/unsorted/viz/core/_Chart.svelte

```diff
@@ -992,7 +992,7 @@
 			});
 		} catch (e) {
 			error = e.message;
-			console.error(e);
+			console.error('\x1b[31m%s\x1b[0m', `Error in ${chartType}: ${e.message}`);
```

Comment:
```suggestion
                         const setTextRed = '\x1b[31m%s\x1b[0m'
			console.error(setTextRed, `Error in ${chartType}: ${e.message}`);
```

The escape sequence here is a fairly "magic" string, if we put it into a variable it's easier to see what is happening at a glance